mods/default/furnace.lua

```diff
@@ -224,6 +223,11 @@ local function furnace_node_timer(pos, elapsed)
 		elapsed = elapsed - el
 	end
 
+	if items_smelt ~= 0 then
```

Comment:
```suggestion
	if items_smelt > 0 then
```

(just a matter of taste, feel free to ignore)